nixos/doc/manual/release-notes/rl-2103.xml

```diff
@@ -168,6 +168,25 @@
       <literal>/var/lib/powerdns</literal> to <literal>/run/pdns</literal>.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The uWSGI server is now built with POSIX capabilities. As a consequence,
```

Comment:
The first time I read this, I was a bit confused.
Capabilities are not standardized and Linux specific.

```suggestion
      The uWSGI server is now started with Linux capabilities instead of running as root. As a consequence,
```

Reply:
You're right: they are not a standard, but based on a POSIX draft nonetheless. Here's what the manual says:

>       No standards govern capabilities, but the Linux capability
>       implementation is based on the withdrawn POSIX.1e draft standard; see
>       ⟨https://archive.org/details/posix_1003.1e-990310⟩.

In any case, "POSIX capabilities" is the term uWSGI documentation use to refer to this:  https://uwsgi-docs.readthedocs.io/en/latest/Capabilities.html

Reply:
ping